### DISCUSSION TITLE
Need to listen on close event after opening camera

### DISCUSSION THREAD

**Discussion by Akhil-coditas** [google/model-viewer#4267] [Q&A]
i am using model viewer in html as below.. such that my section with 100vw gets displayed on top of cube.glft(model which im using) . because my requirement is not to display cube.gltf.

after clicking on "talk to ceo" which is my ar button im applying display:none on the section tag.

so that my camera is opened and section tag disappears.

after opening the camera i get a close button on top right corner by default from model-viewer.. after clicking on close i need to display section again.. how do i listen to close event.? i am not able to find proper docs for it

or is there a better way to handle the above scenario?

```
  <model-viewer
      src="cube.gltf?TED.mp3"
      alt="AR Talk"
      shadow-intensity="1"
      camera-controls
      ar
      ar-modes="webxr"
      id="model-viewer"
      poster="1.png"
      camera-orbit="-4.825deg 85.02deg 3.513m"
      field-of-view="30deg"
    >
      <section id="page1" class="container">
        <div class="profile">
          <p class="profileText">
            A town hall meeting allows CEOs to interact with employees in a
            meaningful way that facilitates strong and productive conversations
          </p>
          <img class="profileImage" src="Profile.jpg" />

        </div>
      </section>
      <div class="progress-bar hide" slot="progress-bar">
        <div class="update-bar"></div>
      </div>
      <button slot="ar-button" id="ar-button" onclick="redirectToAr()">
        Talk to your CEO
      </button>
</model-viewer>
```

**Comment by elalish** [comment]
I believe you're looking for `not-presenting` in https://modelviewer.dev/docs/#entrydocs-augmentedreality-events-arStatus
